src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -97,14 +98,6 @@ public new static RevitDynamoModel Start(IRevitStartConfiguration configuration)
             // where necessary, assign defaults
             if (string.IsNullOrEmpty(configuration.Context))
                 configuration.Context = Core.Context.REVIT_2014;
-            if (string.IsNullOrEmpty(configuration.DynamoCorePath))
-            {
-                var asmLocation = Assembly.GetExecutingAssembly().Location;
-                configuration.DynamoCorePath = Path.GetDirectoryName(asmLocation);
-            }
-
-            if (configuration.Preferences == null)
-                configuration.Preferences = new PreferenceSettings();
```

Comment:
Doing `new PreferenceSettings()` here is as good as doing it in `DynamoModel` base class. Unless `RevitDynamoModel` wants to supply a different implementation, it does not have to do this here (it's making something that is defined in `DynamoCore`, and then pass it back into `DynamoCore`, which is weird). Passing `null` here results in the same `new PreferenceSettings()` call in `DynamoModel`.